awx/ui/src/screens/Instances/InstanceList/InstanceList.js

```diff
@@ -135,6 +140,15 @@ function InstanceList() {
                 onExpandAll={expandAll}
                 qsConfig={QS_CONFIG}
                 additionalControls={[
+                  ...(isK8
```

Comment:
we should also add a check for if the logged in user is a SysAdmin

Reply:
I thought about that...and I'm happy to do that, but this section of the app is only available to sys admins